types/repo/SignedMessageRepo/SignedPostNotificationData.ts

```diff
@@ -91,6 +91,10 @@ export interface PostMaker {
 export interface PostMakerData {
   profile_pic_url?: string;
   username: string;
+  emoji_name: string | null;
+  emoji_code: string | null;
+  color_name: string | null;
+  color_code: string | null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new fields `emoji_name`, `emoji_code`, `color_name`, and `color_code` have been added to the `PostMakerData` interface. Ensure that these fields are properly handled in all places where this interface is used, including proper null checks and default values if necessary. If these fields are optional, consider marking them with a `?` to indicate that they may not always be present. 

```diff
 export interface PostMakerData {
   profile_pic_url?: string;
   username: string;
-  emoji_name: string | null;
-  emoji_code: string | null;
-  color_name: string | null;
-  color_code: string | null;
+  emoji_name?: string | null;
+  emoji_code?: string | null;
+  color_name?: string | null;
+  color_code?: string | null;
 }
```
This change will make it clear that these properties might not always be available, improving code readability and maintainability.


<!-- This is an auto-generated comment by OSS CodeRabbit -->